sources/north-america/us/ct/CT_ECO_Shaded_relief.geojson

```diff
@@ -0,0 +1,468 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "CT_ECO_Shaded_relief",
```

Comment:
Year

Reply:
Does the file name have to be changed as well?

Reply:
I'd recommend doing that, both id and filename